<launch>
   <node pkg="sentry_localization" type="livox_lidar_to_points" name="livox_lidar_to_points_node" output="screen" >
      <param name="livox_in_name" type="string" value="/livox/lidar" />
      <param name="livox_out_name" type="string" value="/livox/points" />
   </node>
</launch>